src/codegen.cpp

```diff
@@ -1134,8 +1134,10 @@ const jl_value_t *jl_dump_function_asm(void *f)
 
 typedef std::map<std::string,std::vector<GlobalVariable*> > logdata_t;
 static logdata_t coverageData;
+// map file => pc => line no
```

Comment:
yeah the comment is wrong now also :-) and the code a bit ugly, I'll do a cleanup pass tomorrow